notifications/event-onerror-default.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Web Notifications Test: notification - onerror (basic)</title>
+<link rel="author" title="Apple Inc." href="http://www.apple.com/">
+<link rel="help" title="5 API" href="http://www.w3.org/TR/notifications/">
+<meta name="flags" content="">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+  <div id="log"></div>
+  <script>
+    if (Notification.permission != "default") {
+      document.getElementById("log").innerHTML = "Please clear your notification settings for web-platform.test and retry.";
```

Comment:
Fixed in https://github.com/w3c/web-platform-tests/commit/a8d5f0411b8816c6c4aadf9896a47f6fbcf3431b